src/actions/user.js

```diff
@@ -20,6 +20,9 @@ export function retrieveUserMemberships() {
                                         membership.profile.has_voip_credentials = (
                                             !!profileData.telavox_username && !!profileData.telavox_password
                                         );
+                                        if (process.env.VOIP_EMBED_URL) {
+                                            membership.profile.voip_embed_url = process.env.VOIP_EMBED_URL;
+                                        }
```

Comment:
Is it necessary to take the long path through `membership.profile`?

Reply:
process.env.VOIP_EMBED_URL needs to be passed in some way from the backend, since the other call integration is at that place it seamed like a good place to keep it, even if not ideal

Reply:
I see that you figured it out using webpack. Basically, the actions can also be run on the frontend, so if the variable is not available on the frontend, using it in `actions/user` is risky. But that should be resolved now with your recent changes.